<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InitialForm</class>
 <widget class="QDialog" name="InitialForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Word Elemination</string>
  </property>
  <widget class="QGroupBox" name="gb">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>251</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLineEdit" name="inputPassword">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputUserName">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblUserName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>用户名</string>
    </property>
    <property name="buddy">
     <cstring>inputUserName</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="lblPassword">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>密码</string>
    </property>
    <property name="buddy">
     <cstring>inputPassword</cstring>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRegister">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>注册</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLogin">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>90</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gb_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>251</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLineEdit" name="inputPassword_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputUserName_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblUserName_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>用户名</string>
    </property>
    <property name="buddy">
     <cstring>inputUserName_2</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="lblPassword_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>密码</string>
    </property>
    <property name="buddy">
     <cstring>inputPassword_2</cstring>
    </property>
   </widget>
   <widget class="QPushButton" name="btnReturnToLogin">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>返回登录</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnConfirmRegister">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>140</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>确认注册</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPassword_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>重复密码</string>
    </property>
    <property name="buddy">
     <cstring>inputPassword_3</cstring>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputPassword_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QRadioButton" name="rPlayer">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>61</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Player</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rDesigner">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>120</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Designer</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
